.github/workflows/build.yaml

```diff
@@ -12,15 +12,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
-      - uses: dtolnay/rust-toolchain@1.75.0
+      - uses: dtolnay/rust-toolchain@1.79.0
```

Comment:
suggestion: This value could be moved into an env variable, so that in the future it only needs to be adjusted in a single place.

Reply:
Please have a look at dfb3affaa9ca681b7e96b67c5b63be471b5b821b